### ISSUE TITLE
DynamoDB "OnDemandThroughput" property reported as invalid 

### ISSUE THREAD

**Issue by axthosarouris** [aws-cloudformation/cfn-lint#3238]
### CloudFormation Lint Version

0.87.2

### What operating system are you using?

Ubuntu

### Describe the bug

DynamoDB "OnDemandThroughput" property reported as invalid 

### Expected behavior

Not reported as invalid

### Reproduction template

```
  EntitiesTable:
    Type: AWS::DynamoDB::Table
    Properties:
      AttributeDefinitions:
        - AttributeName: PK0
          AttributeType: S
        - AttributeName: SK0
          AttributeType: S
        - AttributeName: PK1
          AttributeType: S
        - AttributeName: SK1
          AttributeType: S
        - AttributeName: PK2
          AttributeType: S
        - AttributeName: SK2
          AttributeType: S
      BillingMode: PAY_PER_REQUEST
      OnDemandThroughput:
        MaxReadRequestUnits: 1000
        MaxWriteRequestUnits: 200
      KeySchema:
        - AttributeName: PK0
          KeyType: HASH
        - AttributeName: SK0
          KeyType: RANGE
      GlobalSecondaryIndexes:
        - IndexName: EntriesByExternalId
          Projection:
            ProjectionType: ALL
          KeySchema:
            - AttributeName: PK1
              KeyType: HASH
            - AttributeName: SK1
              KeyType: RANGE
        - IndexName: EntriesByDeletedStatus
          Projection:
            ProjectionType: ALL
          KeySchema:
            - AttributeName: PK2
              KeyType: HASH
            - AttributeName: SK2
              KeyType: RANGE
      PointInTimeRecoverySpecification:
        PointInTimeRecoveryEnabled: True
      TableClass: STANDARD
      StreamSpecification:
        StreamViewType: NEW_AND_OLD_IMAGES
      TimeToLiveSpecification:
        AttributeName: timeToLive
        Enabled: False # Make True when we want to delete
      TableName: !Sub Subscribers-Table-${Suffix}
      Tags:
        - Key: Name
          Value: !Sub '${AWS::StackName}-EntitiesTable'
```

**Comment by kddejong** [comment]
Have you deployed this template into one of these regions?  Since these aren't in the schemas for those regions there is a high likely hood they will fail validation so I want to make sure they are working.

**Comment by axthosarouris** [comment]
> Have you deployed this template into one of these regions? Since these aren't in the schemas for those regions there is a high likely hood they will fail validation so I want to make sure they are working.

I have deployed it in eu-west-1

**Comment by kddejong** [comment]
When running `cfn-lint --regions eu-west-1 -- template.yaml` you got an error with `OnDemandThroughput`?  It looks like that this value is already in the specs for that region.

**Comment by axthosarouris** [comment]
The error was on our side. Thanks for your time?
